include/defines.h

```diff
@@ -275,9 +275,9 @@ enum KART_AI_BEHAVIOURS {
     BEHAVIOUR_NONE = 0,
     BEHAVIOUR_1,
     BEHAVIOUR_HOP,
-    BEHAVIOUR_3,
-    BEHAVIOUR_4,
-    BEHAVIOUR_5,
+    BEHAVIOUR_GO_CENTER,
+    BEHAVIOUR_GO_INNER,
+    BEHAVIOUR_GO_OUTER,
     BEHAVIOUR_NORMAL_SPEED,
     BEHAVIOUR_FAST_SPEED,
     BEHAVIOUR_SLOW_SPEED,
```

Comment:
I wonder if we should replace these with _DRIVE_FAST _DRIVE_SLOW _DRIVE_NORMAL_SPEED

Reply:
in most case this behaviour are advice and not something that cpu should absolutly do